schema/queue/configuration/QueueConfiguration.go

```diff
@@ -32,6 +34,7 @@ type QueueConfigFields struct {
 type QueueConfig struct {
 	QueueConfigFields
 	Filters []MqRouterFilterConfiguration `json:"filters"`
+	Logger  zerolog.Logger
```

Comment:
Why do we have a logger in the configuration? I don't think that it is correct to do